spring-data-cassandra/src/main/java/org/springframework/data/cassandra/core/CassandraTemplate.java

```diff
@@ -91,7 +91,14 @@ public CassandraTemplate(Session session) {
 	 */
 	public CassandraTemplate(Session session, CassandraConverter converter) {
 		setSession(session);
-		setConverter(converter);
+		setConverter(converter == null ? getDefaultCassandraConverter() : converter);
```

Comment:
Not a big deal, but I generally always take a positive perspective first (i.e. innocent until proven guilty ;-), such as...

`setConverter(converter != null ? converter : getDefaultCassandraConverter());`